packages/cli-repl/test/e2e.spec.ts

```diff
@@ -785,11 +786,9 @@ describe('e2e', function() {
     let env: Record<string, string>;
 
     beforeEach(() => {
-      homedir = path.resolve(
-        __dirname, '..', '..', '..', 'tmp', `cli-repl-home-${Date.now()}-${Math.random()}`);
-      env = {
-        ...process.env, HOME: homedir, USERPROFILE: homedir
-      };
+      const homeInfo = setTemporaryHomeDirectory();
+      homedir = homeInfo.homedir;
+      env = homeInfo.env;
```

Comment:
```suggestion
      { homedir, env } = setTemporaryHomeDirectory();
```

Reply:
I also tried like this first, but typescript doesn't like ut :) `Left side of comma operator is unused and has no side effects.`

Reply:
Moved destructing to describe itself. It doesn't look like tests change these values, we can do it once instead of before each test.

Reply:
The problem with that is that then it always runs, even if none of the tests in the file are run, and they won’t be cleaned up by the afterEach hook anymore